build-script.py

```diff
@@ -195,10 +195,10 @@ def generate_gyb_files(
 
     # Clear any *.swift files that are relics from the previous run.
     clear_gyb_files_from_previous_run(
-        SWIFTSYNTAX_DIR, swiftsyntax_destination)
```

Comment:
Found that if an gyb file was removed, the clean missed the verbose.
Should I add this in another PR so it faster get in the main branch?

Reply:
Oh, good catch. 
Yes that would be great if you could open another PR of that!

Reply:
@ahoppen opened in #269